src/ServiceControl.Audit/Infrastructure/RavenDB/EmbeddedRavenDbHostedService.cs

```diff
@@ -57,6 +57,5 @@ public Task StopAsync(CancellationToken cancellationToken)
         }
 
         static readonly ILog Logger = LogManager.GetLogger(typeof(EmbeddedRavenDbHostedService));
-
```

Comment:
I guess this change is not strictly necessary. I'm good though to take it in

Reply:
Tidying up as I go along